enterprise-registry/initial-setup/index.md

```diff
@@ -8,36 +8,21 @@ weight: 5
 
 # On-Premise Installation
 
-CoreOS Enterprise Registry requires five components to operate successfully:
+CoreOS Enterprise Registry requires three components to operate successfully:
 
 - A supported database (MySQL, Postgres)
 - A Redis instance (for real-time events)
-- A config.yaml file
-- An SMTP server
 - The Enterprise Registry image
+- (optional) An SMTP server
 
 
```

Comment:
I think we need a line here saying that you need to have these 3 running, and our wizard will walk you through setting up the rest. Maybe with a screenshot to hint that you'll need things like the IP/port.

Maybe also mention that you can set up stuff like SMTP later on if desired.